### PR TITLE
Button

### PR DESCRIPTION
**Pull request by lannisterrr**
https://user-images.githubusercontent.com/29068538/151652558-fcf07fb1-5e62-4171-87bd-1c7003e15a30.mp4

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+.btn {
+  border: 1px solid var(--secondary-color);
+  outline: none;
+  cursor: pointer;
+  padding: 1rem 1.5rem;
+}
+
+.btn:hover,
+btn:focus {
```
Comment (bhanuprakashj) [inline components/button/button.css]: class selector missing

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  background: transparent;
+}
+
+.btn.outline-danger:hover {
+  background: var(--primary-color);
+  color: var(--secondary-color);
+}
+
+.btn-pill {
+  border-radius: 1000000px;
```
Comment (bhanuprakashj) [inline components/button/button.css:45]: is this really needed?

File: components/button/button.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      <button class="btn btn-danger m-h-3 t-c-1">Danger</button>
+      <button class="btn outline-danger m-h-3 t-c-2">Danger</button>
+      <button class="btn btn-pill f-4 btn-accent m-h-3 t-c-1">Pill</button>
+      <button class="btn btn-custom m-h-3 t-c-1">show</button>
+      <button class="btn btn-danger floating m-8 t-c-1">Floating Button</button>
+      <button class="btn btn-link m-h-3 t-c-3">Link</button>
+      <button class="btn-link m-h-3 t-c-6">
+        <i class="fas fa-shopping-cart f-8"></i>
+      </button>
+      <button class="btn-link m-h-3 t-c-6">
```
Comment (aryan-13) [inline components/button/button.html:27]: These classes m-h-3 and t-c-6 can be given more readable names for better understanding of the code

File: components/button/button.css (hunk 2 above)
Comment (lannisterrr) [reply components/button/button.css:45]: making something custom.

File: components/button/button.html (hunk 3 above)
Comment (lannisterrr) [reply components/button/button.html:27]: these are general classes , it will get lengthy if I give it a long name.

**Pull request merged**